tests/unit_tests/libs_tests/SFML_tests/GraphicsRectangleComponentTests.cpp

```diff
@@ -0,0 +1,19 @@
+/*
+** EPITECH PROJECT, 2022
+** rtype
+** File description:
+** GraphicsRectangleComponentTests
+*/
+
+#include <criterion/criterion.h>
+#include "GraphicsRectangleComponent.hpp"
+
+using namespace ecs;
+
+Test(GraphicsRectangleComponent, test_create_class)
+{
+    GraphicsRectangleComponent a;
+
+    (void)a;
+    cr_assert_eq(1, 1);
```

Comment:
You just want to see if the Rectangle is created ?

Reply:
yes

Reply:
but I will imrpove the test